src/material-experimental/mdc-input/input.spec.ts

```diff
@@ -1081,7 +1081,7 @@ describe('MatMdcInput with forms', () => {
       expect(describedBy).toBe(errorIds);
     }));
 
-    it('should not set `aria-invalid` to true if the input is empty', fakeAsync(() => {
+    it('should not set `aria-invalid` if the input is empty', fakeAsync(() => {
```

Comment:
Looking at this test in more detail.

Reply:
Edge case:  This test tests an input where `Validators.required` is added to the form field but the actual `input` with the `matInput` directive does not have the `required` attribute.

In this case, the `aria-invalid` attribute should be `true` since `aria-required` is _not_ present on the element.  This is the exact edge case @zelliott was trying to catch!